qcodes/utils/dataset/doNd.py

```diff
@@ -143,18 +165,36 @@ def do1d(
         write_period: The time after which the data is actually written to the
             database.
         additional_setpoints: A list of setpoint parameters to be registered in
-            the measurement but not scanned.
+            the measurement but not scanned. TODO it should be enforced that
+            these are scalar parameters
```

Comment:
should TODO now be gone?

Reply:
It's not fixed but it does not really have anything to do with this pr